Gems/ROS2/Code/Source/Georeference/GNSSFormatConversions.cpp

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Could we use some of header-only open source libraries for format conversion instead (to maintain less code?)
Meaning projects like https://github.com/chrberger/WGS84toCartesian etc.
See https://github.com/topics/wgs84?l=c%2B%2B.

To be discussed.